addon/mixins/application-route-mixin.js

```diff
@@ -66,8 +66,19 @@ export default Mixin.create({
     @default 'index'
     @public
   */
-  routeAfterAuthentication: computed(function() {
-    return Configuration.routeAfterAuthentication;
+  routeAfterAuthentication: computed({
+    get() {
+      if (this.hasOwnProperty('routeAfterAuthenticationOverride')) {
+        return this.routeAfterAuthenticationOverride;
+      } else {
+        return Configuration.routeAfterAuthentication;
+      }
+    },
+
+    set(key, value) {
+      this.routeAfterAuthenticationOverride = value;
+      return value;
+    }
   }),
```

Comment:
Could this not be replaced with

```js
routeAfterAuthentication: Configuration.routeAfterAuthentication,
```

simply?  I can't remember why this was made a CP in the first place…